### DISCUSSION TITLE
Suggestion: City Buildings/Homes?

### DISCUSSION THREAD

**Discussion by okienative99** [niels747/2D-Weather-Sandbox#34] [Ideas]
Good day! 

I am curious to know if there are any thoughts on adding buildings/homes to the simulation. I live in a metropolitan area, and we get a lot of surface heating from the Urban Island Effect, which causes heat in a concentrated area versus surrounding areas. I notice when storms go over the area, they tend to grow quite rapidly. I would love to see this happen in the simulation; thanks!

**Comment by niels747** [comment]
Yes, great idea. I think I will implement it with a texture at the surface just like the trees. The height Limit will be 40 meters. But I could increase that somehow. 
Urban areas do indeed heat faster because of low albedo and low evaporation. I will add those factors in the simulation. Don't have much time though, so it will be next year.

**Reply by okienative99** [reply]
@niels747 Great! Thanks for getting back to me fairly quickly.

**Reply by MET905** [reply]
I love the Update, one thing id love to see is the option to add a lake or an ocean, as both masses of water are not always the same temperature. doing that would allow me to build a pretty good simulation of the golf of Mexico's storm systems that impact the great lakes. i seen a request for this awhile ago and i believe it would be awesome to add!

**Comment by niko-bie** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Reply by Westerstaad** [reply]
bing chilling

**Comment by JustLearninglol** [comment]
Cities should be able to emit smog too

**Comment by FerdinandoLM** [comment]
@niels747 this comment section has been get targeted by spam.
